### ISSUE TITLE
Do you think it would be possible to runn it at lower prezion to make it faster and requier less vram? or would the quallaty suffer to much under that?

### ISSUE THREAD

**Issue by Manni1000** [VectorSpaceLab/OmniGen#9]

**Comment by BitPhinix** [comment]
It would probably work just fine, but there isn't even a released model yet, so I guess you can find that out yourself.

**Comment by staoxiao** [comment]
We have not done much optimization for inference at the moment, but we believe there is significant room for improvement in OmniGen's inference efficiency. We also welcome contributions from the community.

**Comment by Manni1000** [comment]
yes that would be cool. just a fun question nothing like a request or something. what do you think about stronger latent space compression? this could speed up genration time for future models a lot. würstchen3(stable cascade) https://github.com/Stability-AI/StableCascade did it and now the pixart team at nvidia did it  https://github.com/mit-han-lab/efficientvit . btw the model is very cool thank you for open sourcing it!
